README.md

```diff
@@ -118,6 +203,26 @@ system to be trustworthy and predictable in its performance and user experience.
 
 ## S
 
+### Span
+
+A span is a single unit of work in a distributed system, often represented by a trace. Spans can be used to identify the performance of individual components in a system.
+
+### Serverless
+
+Serverless is a model where a cloud provider manages the infrastructure and automatically scales resources, allowing developers to focus on code. This model is becoming increasingly popular for building and deploying applications.
+
+### SRE
+
+SRE (Site Reliability Engineering) is a discipline focused on improving and maintaining the reliability of systems. SREs use a data-driven approach to manage the performance and availability of systems.
+
+### Service Discovery
+
+Service discovery is the process of automatically identifying and locating available services in a distributed system. Service discovery is important for managing the complexity of distributed systems.
```

Comment:
```suggestion
Service Discovery is automatically identifying and locating available services in a distributed system. Service Discovery is essential for managing the complexity of distributed systems because of the dynamic and ephemeral nature of the systems.
```